### DISCUSSION TITLE
quiet docker destroy

### DISCUSSION THREAD

**Discussion by emteeoh** [kreuzwerker/terraform-provider-docker#221] [Q&A] [answered]
Is there a way to configure a terraform pipeline so that when I do a terraform destroy, I don't get an error about not removing an docker image from cache, without forcing a destroy? I'm happy to keep the cache. In fact, I want to keep the cache.

**Comment by mavogel** [comment]
Hi, I do not fully understand the issue. Do you have an example for me? :)

**Comment by suzuki-shunsuke** [comment, marked as answer]
Maybe `keep_locally` option is useful.
https://registry.terraform.io/providers/kreuzwerker/docker/latest/docs/resources/image

**Reply by emteeoh** [reply]
Yeah, that's what I'm looking for. I'm not sure how I overlooked it reading the docs on my own. Thanks!
